### PR TITLE
[Issue-372] prepare makefile to build job

### PR DESCRIPTION
**Pull request by yurkov-anton**
### Issue dell/csi-baremetal#372

Add variables.mk with versions.
Add image path
Add make version

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
+include variables.mk
+
 # Image URL to use all building/pushing image targets
 # TODO refactor after csi-baremetal-operator renaming https://github.com/dell/csi-baremetal/issues/341
-IMG ?= controller:latest
+IMG ?= ${REGISTRY}/csi-barmetal-operator:${TAG}
```
Comment (mishoyama) [inline Makefile:5]: Typo in the name.

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,9 @@
+include variables.mk
+
 # Image URL to use all building/pushing image targets
 # TODO refactor after csi-baremetal-operator renaming https://github.com/dell/csi-baremetal/issues/341
```
Comment (safronovD) [inline Makefile:4]: Please, remove this TODO and close the issue

File: Makefile (hunk 1 above)
Comment (yurkov-anton) [reply Makefile]: Fixed.

File: Makefile (hunk 2 above)
Comment (yurkov-anton) [reply Makefile]: Fixed.

**Review by mishoyama — APPROVED** [review]

**Review by safronovD — APPROVED** [review]

**Pull request merged**